src/amo/searchUtils.js

```diff
@@ -26,6 +26,7 @@ export const paramsToFilter = {
   author: 'author',
   category: 'category',
   color: 'color',
+  ...generateThresholdParams('created'),
```

Comment:
Why spreading here and not at the bottom of the object?

Reply:
I'm following the existing convention and ordering keys alphabetically. We already had `...generateThresholdParams('ratings'),` placed after `q: 'query',`.